pylint/test/unittest_checker_imports.py

```diff
@@ -116,14 +116,14 @@ def test_wildcard_import_init(self):
     def test_wildcard_import_non_init(self):
         here = os.path.abspath(os.path.dirname(__file__))
         path = os.path.join(
-            here, 'regrtest_data', 'init_wildcard', '__init__.py')
+            here, 'regrtest_data', 'awesome_module.py')
```

Comment:
Wonder why this was updated?

Reply:
The content of `regrtest_data/init_wildcard/__init__.py` is `from awesome_module import *`, later on in the test it parses the module as if it was named `awesome_module`, which causes an unexpected self-import warning.  (It wasn't emitted previously due to wildcard imports being ignored.)  It might be an idea to rename these files, "awesome_module" is not very descriptive of what's being tested.

Reply:
Sounds good, thanks for the explanation. We can definitely rename modules.